lib/iex/test/iex/helpers_test.exs

```diff
@@ -79,6 +79,25 @@ defmodule IEx.HelpersTest do
 
     assert "@opaque t()\n" = capture_io(fn -> t HashDict.t end)
     assert capture_io(fn -> t HashDict.t end) == capture_io(fn -> t HashDict.t/0 end)
+
+    filename = "typesample.ex"
+    with_file filename, module_with_typespecs, fn ->
+      c(filename)
+      assert capture_io(fn -> t TypeSample.id_with_desc/0 end) == """
+* TypeSample
```

Comment:
Let's put this on same level with `assert`.
By using `"""` we can align content as deed as last `"""` goes, for example https://github.com/Maikon/elixir/blob/show-type-docs-with-t1/lib/iex/test/iex/helpers_test.exs#L56-L58.